addons/web/static/src/scss/form_view.scss

```diff
@@ -564,7 +564,7 @@
     }
 
     // Form fields
-    .o_field_widget {
+    .o_field_widget:not(.oe_stat_button) {
```

Comment:
The css of the form view is tricky to change. This is quite risky as you increase the priority of a rule impacting all widgets. Set a 0 margin-bottom on `oe_stat_button` instead (where the rules of `oe_stat_button` are defined), and add a comment like `// If the button comes from a field`